examples/end-to-end-session-based/02-End-to-end-session-based-with-Yoochoose-PyT.ipynb

```diff
@@ -112,10 +112,10 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    table = TensorTable.from_df(df.iloc[:100])
</pre><p>can you please add a new cell and add these two lines ( we need my PR to be merged first for the top_k part works ): </p><p> </p><p> </p><p> <code></code><code></p><p>Let's now add a top_k parameter to model so that we can return the top k item ids with the highest scores when we serve the model on Triton Inference Server.</p><p> </p><p>model.top_k = 28</p><p>model.eval()</p><p></code><code></code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/680/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
> Line #2.    table = TensorTable.from_df(df.iloc[:100])
> can you please add a new cell and add these two lines ( we need my PR to be merged first for the top_k part works ):
> 
> `Let's now add a top_k parameter to model so that we can return the top k item ids with the highest scores when we serve the model on Triton Inference Server.`
> 
> ` `
> 
> `model.top_k = 28`
> 
> `model.eval()`
> 
> 
> _Reply via [ReviewNB](https://app.reviewnb.com/NVIDIA-Merlin/Transformers4Rec/pull/680/discussion/#comment-1177855139)_

you can set top_k `10` if you want. I set it to `28` just as an example #468